tools/wpt/browser.py

```diff
@@ -407,6 +407,12 @@ def install_webdriver(self, dest=None):
     def version(self, binary):
         return None
 
+class ChromeWebdriver(Chrome):
```

Comment:
This capitalization bugged me so I grepped a bit and sent https://github.com/web-platform-tests/wpt/pull/12767. Can you do `sed -i '' s/Webdriver/WebDriver/g` for the changes in this PR?